morphocluster/frontend/src/views/File.vue

```diff
@@ -0,0 +1,93 @@
+
```

Comment:
Please merge functionality into `Files.vue` (analogously to api.py:get_filesystem_entry).